src/processTargets/processTargets.ts

```diff
@@ -144,7 +146,17 @@ function processVerticalRangeTarget(
       anchorTarget.contentRange.end.character
     );
 
-    results.push(anchorTarget.withContentRange(contentRange));
+    if (anchorTarget instanceof PositionTarget) {
+      results.push(anchorTarget.withContentRange(contentRange));
+    } else {
+      results.push(
+        new PlainTarget({
```

Comment:
Do we want to use a `WeakTarget` instead? Might be nice to clean up white space and do weak expansion, given these are more like a bunch of marks in my mind

Reply:
You do make a point but at the same time I do feel that the behavior of sliced targets right now is a fixed range that expands to a column and if I tell it to slice a specific column I don't want anything unexpected to change that range. This is a real edge case but I believe that plain target is more correct.

Reply:
Ok we have pretty different expectations of slice semantics. Let's leave as plain for now but aim to discuss

Will you miss your rigid block slice semantics if we move to post-slice modifier application? If so then maybe we need two different kinds of slice?

Reply:
Agreed we definitely need to iron this one out 

I guess in that case I can just use just/raw to get the same results. But that is definitely something we need to think about.

Reply:
I'm not sure "just" helps you here. What exactly were you planning to say?

Fwiw we could just add a new range modifier called "strict slice"

Reply:
"take line (just air slice bat)"
I have no idea how we should do the grammar so that line should run after but just should run before. hmm

Because I don't think we should make it so all modifiers ran after the slice range.

Reply:
Ok let's plan to discuss

Reply:
Definitely

Reply:
We could change the grammar a bit
"take line slice air past bat"
"take slice line air past bat"

Reply:
yeah worth thinking bout.  Fwiw I updated https://github.com/cursorless-dev/cursorless/issues/193 in prep for this discussion.  Lmk if that formalism makes sense

Reply:
Works for me